mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -155,13 +155,13 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 			COMMENT The x-coordinate on the unit square
 		ARG 2 deltaY
 			COMMENT The y-coordinate on the unit square
-		ARG 4 val00
```

Comment:
Personally, I think these should be left as-is because they're used in other places that don't need to follow this convention

Reply:
This would make the parameters clearer there too though. And they mean basically the same thing.

Reply:
Took another look at it and I get what you're saying

Reply:
i approce these param changes as well.